### ISSUE TITLE
Techdocs CLI fails to import a module upon hitting serve

### ISSUE THREAD

**Issue by swist** [backstage/techdocs-cli#9]
I have an mkdoc setup in a repo (external to my backstage repo). I'm trying to run techdocs-cli from a folder which contains mkdocs documentation but i get the following:

```
$ npx run @techdocs/cli serve
npx: installed 5 in 1.147s
Watching /Users/swistak/code/notifications and all sub-directories not excluded by your .gitignore. Will not monitor dotfiles.

Starting: @techdocs/cli serve
internal/modules/cjs/loader.js:969
  throw err;
  ^

Error: Cannot find module '/Users/swistak/code/notifications/@techdocs/cli'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:966:15)
    at Function.Module._load (internal/modules/cjs/loader.js:842:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}


```

I'm not sure if the repo containing docs needs to be an npm repo? Why? It appears that there's no `dist` or `src` folder in the cli uploaded to npm

**Comment by OrkoHunter** [comment]
Hi @swist ! Thank you very much for reporting. We found out that the latest release of `tecdocs-cli` is broken.

While we work on fixing this, you can try a version [0.1.1-alpha.23](https://www.npmjs.com/package/@techdocs/cli/v/0.1.1-alpha.23) or older. Apologies! We'll let you know when we release a newer version that is working.

**Comment by OrkoHunter** [comment]
Note: The fix we have in mind is currently blocked by this issue https://github.com/backstage/backstage/issues/2707

**Comment by hooloovooo** [comment]
Fixed in 0.1.7.
